enginetest/queries/insert_queries.go

```diff
@@ -1116,6 +1116,9 @@ var InsertScripts = []ScriptTest{
 			"CREATE TABLE t2(id varchar(100) DEFAULT (uuid()));",
 			"CREATE TABLE t3(a int DEFAULT '1', b int default (2 * a));",
 			"CREATE TABLE t4(c0 varchar(10) null default 'c0', c1 varchar(10) null default 'c1');",
+			// MySQL allows the current_timestamp() function to NOT be in parens when used as a default
+			// https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html
+			"CREATE TABLE t5(c0 varchar(100) DEFAULT (repeat('_', 100)), c1 datetime DEFAULT current_timestamp());",
```

Comment:
the only things i'm not sure if i saw tested, were:
- execution tests with parenthesized datetime funcs with datetime types (seems like this would still be valid?)
- alter schema tests going to/from valid and invalid timestamp func defaults

Reply:
Yup yup, now/current_timestamp can also be used in parentheses. I know I saw some existing tests for these. Alter schema tests is a good idea. I'll take another look at those and how our coverage looks before checking this in.